Tests/FluentAssertions.Shared.Specs/DateTimeAssertionSpecs.cs

```diff
@@ -792,6 +792,47 @@ public void Should_fail_with_descriptive_message_when_asserting_datetime_has_sec
                 .WithMessage("Expected second 1 because we want to test the failure message, but found 0.");
         }
 
+        [TestMethod]
+        public void Should_succeed_when_asserting_datetime_is_on_same_date()
```

Comment:
Please write the specs using the same layout and naming conventions as https://github.com/dennisdoomen/fluentassertions/blob/master/Tests/FluentAssertions.Shared.Specs/BasicEquivalencySpecs.cs#L33